manager/api/beans/src/main/java/io/apiman/manager/api/beans/apis/UpdateApiVersionBean.java

```diff
@@ -46,9 +46,9 @@
     private Boolean parsePayload;
     private Boolean publicAPI;
     private Boolean disableKeysStrip;
-    private Set<ApiPlanBeanDto> plans;
+    private @Valid Set<UpdateApiPlanDto> plans;
     private String extendedDescription;
-    private Boolean exposeInPortal;
+    private DiscoverabilityLevel publicDiscoverability;
```

Comment:
Need to be consistent and decide whether we call it `publicDiscoverability` when the discoverability attaches to a public API, and if so, do it everywhere.